### PR TITLE
When user running ss-local but not input any argument, ss-local will try to use ./config(if exist).

### PR DESCRIPTION
**Pull request by JohnnySun**
When user running ss-local but not input any argument, ss-local will try to use ./config(if exist).

### CODE REVIEW

**Comment by madeye** [comment]
Squash your commits first: https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

File: src/local.c [hunk 1]
```diff
@@ -963,12 +963,17 @@ int main(int argc, char **argv)
...
+            conf_path = DEFAULT_CONF_PATH;
+        }
+    }
     if (conf_path != NULL) {
         jconf_t *conf = read_jconf(conf_path);
         if (remote_num == 0) {
             remote_num = conf->remote_num;
             for (i = 0; i < remote_num; i++) {
-                remote_addr[i] = conf->remote_addr[i];
+                 remote_addr[i] = conf->remote_addr[i];
```
Comment (madeye) [inline src/local.c]: Fix the indent.

File: src/local.c [hunk 2]
```diff
@@ -1001,7 +1006,7 @@ int main(int argc, char **argv)
          * the user an error message if setrlimit(2) fails
          */
         if (nofile) {
-            if (verbose) {
+            if (verbose) {
```
Comment (madeye) [inline src/local.c]: Trim the white space.

**Comment by JohnnySun** [comment]
我已经建立了一个新分支，本地合并了这些commit。提交时候还是显示5个。

**Comment by madeye** [comment]
You need to resubmit this pull request with the squashed commit.

**Comment by madeye** [comment]
A dirty way is to delete your fork branch, fork again, apply the final patch and then submit the pull request.

**Comment by JohnnySun** [comment]
see #280

**Pull request closed without merging**